### ISSUE TITLE
Building sounds

### ISSUE THREAD

**Issue by Helkus** [Facepunch/rust#134]
No sound when building or upgrading anymore

**Comment by garrynewman** [comment]
Yep this is on me - the effect is being received before the building spawns on the client - so it doesn't know where to spawn. I'll come up with a solution for it.

**Comment by arehberg** [comment]
We could probably just put a SoundPlayer on the building part so it'll play the sound when it spawns. Then we won't even need to send the effect over the network.

**Comment by garrynewman** [comment]
Wouldn't it play when you join a server and all the building parts spawn

**Comment by arehberg** [comment]
Right now, yeah. That should be pretty easy to solve though. I just need to add a bool to prevent the SoundPlayer from playing immediatly (which I plan to do for some other stuff already). Then I'd assume we can just switch that off for the initial batch of building parts

**Comment by garrynewman** [comment]
Seems like adding a sound to each building block then messing with a bunch of bools and hoping the sound doesn't go off when it's not meant to is a worse solution than just fixing effects not spawning properly.
